devel/setup-e2e-deps.sh

```diff
@@ -37,34 +37,22 @@ echo "Installing cert-manager into the cluster..."
 check_bazel
 bazel build --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 //devel/addon/...
 
-EXIT=0
-
 echo "Installing sample-webhook into the cluster..."
-"${SCRIPT_ROOT}/addon/samplewebhook/install.sh" &
+"${SCRIPT_ROOT}/addon/samplewebhook/install.sh" 
 
 echo "Installing bind into the cluster..."
-"${SCRIPT_ROOT}/addon/bind/install.sh" &
+"${SCRIPT_ROOT}/addon/bind/install.sh" 
 
 echo "Installing pebble into the cluster..."
-"${SCRIPT_ROOT}/addon/pebble/install.sh" &
+"${SCRIPT_ROOT}/addon/pebble/install.sh" 
 
 echo "Installing ingress-nginx into the cluster..."
-"${SCRIPT_ROOT}/addon/ingressnginx/install.sh" &
+"${SCRIPT_ROOT}/addon/ingressnginx/install.sh" 
 
 echo "Loading vault into the cluster..."
-"${SCRIPT_ROOT}/addon/vault/install.sh" &
+"${SCRIPT_ROOT}/addon/vault/install.sh" 
 
 echo "Installing sample-external-issuer into the cluster..."
-"${SCRIPT_ROOT}/addon/sample-external-issuer/install.sh" &
-
-# Wait for all background jobs to finish and exit with non-zero if any of them fail
-# See https://stackoverflow.com/a/515170/919436
-for job in $(jobs -p); do
-    wait $job || let "EXIT+=1"
-done
-
-if [[ "$EXIT" > 0 ]]; then
-    echo "ERROR: ${EXIT} setup jobs failed. Check logs above for details."
-fi
+"${SCRIPT_ROOT}/addon/sample-external-issuer/install.sh" 
 
-exit $EXIT
+exit 0
```

Comment:
@wallrj is it fine if we return with an exit code of 0 always (even when some tests fail)? Cause we would see the logs for the failed tests anyways.

Or should I add back the code where we wait for jobs to finish so that we can keep a track of the number of tests failed?

Reply:
You can remove the `exit` line, because the script is configured with `-o errexit` (see top of script) which makes it exit with `> 1` if any of the statements fail. See https://elder.dev/posts/safer-bash/